### PR TITLE
test ersrating when extract from csv file

### PR DESCRIPTION
**Pull request by vadjr**
Add testing for ERSRATING when value is empty

### CODE REVIEW

File: etl/src/energuide/dwelling.py [hunk 1]
```diff
@@ -171,6 +180,7 @@ def __init__(self, *,
                  heated_floor_area: heated_floor_area.HeatedFloorArea,
                  water_heatings: typing.List[water_heating.WaterHeating],
                  ventilations: typing.List[ventilation.Ventilation],
+                 ers_rating: typing.Optional[int]        #vadim
```
Comment (JasonMWhite) [inline etl/src/energuide/dwelling.py]: Please remove these comments

File: etl/src/energuide/dwelling.py [hunk 2]
```diff
@@ -184,6 +194,7 @@ def __init__(self, *,
         self._heated_floor_area = heated_floor_area
         self._ventilations = ventilations
         self._water_heatings = water_heatings
+        self._ers_rating = ers_rating        #vadim
```
Comment (JasonMWhite) [inline etl/src/energuide/dwelling.py]: This one too

File: etl/src/energuide/dwelling.py [hunk 3]
```diff
@@ -285,6 +303,7 @@ def __init__(self, *,
         self._forward_sortation_area = forward_sortation_area
         self._evaluations = evaluations
 
+
```
Comment (JasonMWhite) [inline etl/src/energuide/dwelling.py]: remove this extra newline

File: etl/src/energuide/dwelling.py [hunk 4]
```diff
@@ -277,6 +294,7 @@ def __init__(self, *,
                  city: str,
                  region: Region,
                  forward_sortation_area: str,
+
```
Comment (JasonMWhite) [inline etl/src/energuide/dwelling.py]: remove this extra newline
